### ISSUE TITLE
Random app crashes

### ISSUE THREAD

**Issue by kruleucc** [jellyfin/jellyfin-androidtv#2936]
### Describe the bug

So, I have no exact steps to reproduce this crash.
It sometimes happens when turning the tv on and starting the app, sometimes it happens after pausing the playback for a while, it is really random.

### Logs

This is the server log: https://pastebin.com/Erm4q3UY
And this is the client log: https://pastebin.com/MxYN3Hhu

### Application version

0.15.11

### Where did you install the app from?

Google Play

### Device information

Xiaomi MiTV P1 43"

### Android version

Android TV 10

### Jellyfin server version

10.8.9

**Comment by nielsvanvelzen** [comment]
In addition to above comment you might want to test it in the recently released 0.16 beta, although it looks like an issue in the server.

**Comment by kruleucc** [comment]
> In addition to above comment you might want to test it in the recently released 0.16 beta, although it looks like an issue in the server.

I updated to beta, issue is still here. Will answer back when the server owner updates the server.

**Comment by kruleucc** [comment]
So, the server owner updated the server, I updated to the beta release, and the crashes still happened, but the logs are different I think. 
Here's the log:

---
client: Jellyfin for Android TV
client_version: 0.16.0-beta.1
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.IllegalArgumentException: No view found for id 0x7f0b026e (org.jellyfin.androidtv:id/rowsFragment) for fragment RowsSupportFragment{4c589d7} (465c38c0-4053-4e31-91fb-358b3aee6f9d id=0x7f0b026e)
	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:547)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1839)
	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782)
	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:565)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:932)

```  
***Logcat***: 
```log
08-05 00:25:51.091 I/jellyfin-apiclient(14719): Response received from: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:25:52.255 D/CustomPlaybackOverlayFragment(14719): rendering subtitle event: 652 (pos 1252593 start 1252251 end 1253653)
08-05 00:25:53.765 D/CustomPlaybackOverlayFragment(14719): rendering subtitle event: 653 (pos 1254097 start 1253686 end 1255822)
08-05 00:25:53.898 D/jellyfin-apiclient(14719): Adding request to queue: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:25:54.080 I/jellyfin-apiclient(14719): Response received from: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:25:55.777 D/CustomPlaybackOverlayFragment(14719): rendering subtitle event: 654 (pos 1256112 start 1255855 end 1256889)
08-05 00:25:56.787 D/CustomPlaybackOverlayFragment(14719): rendering subtitle event: 655 (pos 1257127 start 1256923 end 1259058)
08-05 00:25:56.901 D/jellyfin-apiclient(14719): Adding request to queue: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:25:57.052 I/jellyfin-apiclient(14719): Response received from: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:25:58.795 D/CustomPlaybackOverlayFragment(14719): rendering subtitle event: 656 (pos 1259134 start 1259091 end 1260460)
08-05 00:25:59.901 D/jellyfin-apiclient(14719): Adding request to queue: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:26:00.093 I/jellyfin-apiclient(14719): Response received from: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:26:00.304 D/CustomPlaybackOverlayFragment(14719): rendering subtitle event: 657 (pos 1260639 start 1260493 end 1262161)
08-05 00:26:02.315 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:02.817 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:02.904 D/jellyfin-apiclient(14719): Adding request to queue: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:26:03.056 I/jellyfin-apiclient(14719): Response received from: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:26:03.319 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:03.822 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:04.324 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:04.826 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:05.328 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:05.830 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:05.905 D/jellyfin-apiclient(14719): Adding request to queue: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:26:06.052 I/jellyfin-apiclient(14719): Response received from: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:26:06.334 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:06.837 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:07.339 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:07.842 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:08.344 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:08.847 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:08.906 D/jellyfin-apiclient(14719): Adding request to queue: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:26:09.061 I/jellyfin-apiclient(14719): Response received from: https://jellyfin.example.dev/Sessions/Playing/Progress
08-05 00:26:09.201 V/AudioTrack(14719): stop(439): prior state:STATE_ACTIVE
08-05 00:26:09.201 D/AudioTrack(14719): stop(439): called with 60972032 frames delivered
08-05 00:26:09.349 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:09.852 D/CustomPlaybackOverlayFragment(14719): subtitles handled in 2 iterations
08-05 00:26:09.961 D/PlaybackController(14719): On Completion fired
08-05 00:26:09.961 D/PlaybackController(14719): stop called at 1270186
08-05 00:26:10.112 I/jellyfin-apiclient(14719): ReportPlaybackStopped: Item ac178466-1be1-b8ea-70a7-1f971d334ec8, Ticks: 12701860000
08-05 00:26:10.112 D/jellyfin-apiclient(14719): Adding request to queue: https://jellyfin.example.dev/Sessions/Playing/Stopped
08-05 00:26:10.115 D/PlaybackController(14719): Moving to next queue item. Index: 1
08-05 00:26:10.117 D/NavigationRepositoryImpl(14719): Navigating to Fragment(fragment=class org.jellyfin.androidtv.ui.playback.nextup.NextUpFragment (Kotlin reflection is not available), arguments=Bundle[{item_id=78072184-0ca2-6a62-50f3-d8e41ac79e48}]) (via navigate function)
08-05 00:26:10.127 D/PlaybackController(14719): stop called at 1270186
08-05 00:26:10.140 V/AudioTrack(14719): flush(439): prior state:STATE_STOPPED
08-05 00:26:10.142 V/AudioTrack(14719): stop(439): prior state:STATE_FLUSHED
08-05 00:26:10.146 V/AudioTrack(14719): stop(439): prior state:STATE_FLUSHED
08-05 00:26:10.148 V/AudioTrack(14719): ~AudioTrack(439), releasing session id 1201 from 14719 on behalf of 14719
08-05 00:26:10.241 D/SurfaceUtils(14719): disconnecting from surface 0x7614e008, reason disconnectFromSurface
08-05 00:26:10.289 I/ExoPlayerImpl(14719): Release c4026ec [ExoPlayerLib/2.19.0] [croods, MiTV-MOOQ0, Xiaomi, 29] [goog.exo.core, goog.exo.ui, goog.exo.exoplayer, goog.exo.decoder, goog.exo.datasource, goog.exo.extractor, goog.exo.ffmpeg]
08-05 00:26:10.304 D/CustomPlaybackOverlayFragment(14719): Stopping!
08-05 00:26:10.320 D/jellyfin-apiclient(14719): Adding request to queue: https://jellyfin.example.dev/Users/14b8fc0b-6c85-49db-875e-0400de50b3ef/Items/ac178466-1be1-b8ea-70a7-1f971d334ec8?format=json
08-05 00:26:10.337 D/LegacyMediaManager(14719): Added event listener.  Total listeners: 2
08-05 00:26:10.346 D/Compose Focus(14719): Owner FocusChanged(true)
08-05 00:26:10.356 I/org.jellyfin.sdk.api.client.KtorClient(14719): GET https://jellyfin.example.dev/Users/14b8fc0b-6c85-49db-875e-0400de50b3ef/Items/78072184-0ca2-6a62-50f3-d8e41ac79e48
08-05 00:26:10.398 D/Compose Focus(14719): Owner FocusChanged(false)
08-05 00:26:10.441 D/Compose Focus(14719): Owner FocusChanged(true)
08-05 00:26:10.444 D/Compose Focus(14719): Owner FocusChanged(false)
08-05 00:26:10.453 D/AndroidRuntime(14719): Shutting down VM
--------- beginning of crash
08-05 00:26:10.457 E/AndroidRuntime(14719): FATAL EXCEPTION: main
08-05 00:26:10.457 E/AndroidRuntime(14719): Process: org.jellyfin.androidtv, PID: 14719
08-05 00:26:10.457 E/AndroidRuntime(14719): java.lang.IllegalArgumentException: No view found for id 0x7f0b026e (org.jellyfin.androidtv:id/rowsFragment) for fragment RowsSupportFragment{4c589d7} (465c38c0-4053-4e31-91fb-358b3aee6f9d id=0x7f0b026e)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:547)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1839)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:565)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at android.os.Handler.handleCallback(Handler.java:883)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at android.os.Handler.dispatchMessage(Handler.java:100)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at android.os.Looper.loop(Looper.java:214)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at android.app.ActivityThread.main(ActivityThread.java:7356)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at java.lang.reflect.Method.invoke(Native Method)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
08-05 00:26:10.457 E/AndroidRuntime(14719): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:932)
08-05 00:26:10.463 E/ACRA    (14719): ACRA caught a IllegalArgumentException for org.jellyfin.androidtv
08-05 00:26:10.463 E/ACRA    (14719): java.lang.IllegalArgumentException: No view found for id 0x7f0b026e (org.jellyfin.androidtv:id/rowsFragment) for fragment RowsSupportFragment{4c589d7} (465c38c0-4053-4e31-91fb-358b3aee6f9d id=0x7f0b026e)
08-05 00:26:10.463 E/ACRA    (14719): 	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:547)
08-05 00:26:10.463 E/ACRA    (14719): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:272)
08-05 00:26:10.463 E/ACRA    (14719): 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1943)
08-05 00:26:10.463 E/ACRA    (14719): 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1839)
08-05 00:26:10.463 E/ACRA    (14719): 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1782)
08-05 00:26:10.463 E/ACRA    (14719): 	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:565)
08-05 00:26:10.463 E/ACRA    (14719): 	at android.os.Handler.handleCallback(Handler.java:883)
08-05 00:26:10.463 E/ACRA    (14719): 	at android.os.Handler.dispatchMessage(Handler.java:100)
08-05 00:26:10.463 E/ACRA    (14719): 	at android.os.Looper.loop(Looper.java:214)
08-05 00:26:10.463 E/ACRA    (14719): 	at android.app.ActivityThread.main(ActivityThread.java:7356)
08-05 00:26:10.463 E/ACRA    (14719): 	at java.lang.reflect.Method.invoke(Native Method)
08-05 00:26:10.463 E/ACRA    (14719): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
08-05 00:26:10.463 E/ACRA    (14719): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:932)
08-05 00:26:10.463 D/ACRA    (14719): Building report
08-05 00:26:10.470 D/ACRA    (14719): Starting collectors with priority FIRST
08-05 00:26:10.472 D/ACRA    (14719): Calling collector org.acra.collector.LogCatCollector
08-05 00:26:10.472 D/ACRA    (14719): Calling collector org.acra.collector.DropBoxCollector
08-05 00:26:10.472 D/ACRA    (14719): Collector org.acra.collector.DropBoxCollector completed
08-05 00:26:10.473 D/ACRA    (14719): Calling collector org.acra.collector.StacktraceCollector
08-05 00:26:10.473 D/ACRA    (14719): Collector org.acra.collector.StacktraceCollector completed
08-05 00:26:10.489 D/ACRA    (14719): Retrieving logcat output (buffer:default)...

```  

### App information

***App version***: `0.16.0-beta.1` (`160001`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"croods","BOOTLOADER":"123456789","BRAND":"Xiaomi","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"croods","DISPLAY":"QTG3.200305.006.5135 release-keys","FINGERPRINT":"Xiaomi\/croods\/croods:10\/QTG3.200305.006\/5135:user\/release-keys","HARDWARE":"m7632","HOST":"android-build","ID":"QTG3.200305.006","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Xiaomi","MODEL":"MiTV-MOOQ0","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"croods","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1662307019000,"TYPE":"user","UNKNOWN":"unknown","USER":"nobody","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","FIRST_SDK_INT":29,"INCREMENTAL":"5135","PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"10","RESOURCES_SDK_INT":29,"SDK":"29","SDK_INT":29,"SECURITY_PATCH":"2022-08-05"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":160001,"VERSION_NAME":"0.16.0-beta.1"}
```  

### Device information

***Android version***: `10`  
***Device brand***: `Xiaomi`  
***Device product***: `croods`  
***Device model***: `MiTV-MOOQ0`  

### Crash information

***Start time***: `2023-08-04T23:08:28.016+02:00`  
***Crash time***: `2023-08-05T00:26:10.609+02:00`

**Comment by nielsvanvelzen** [comment]
That specific crash is #2931 which is already fixed for the next beta!
